hack/common.sh

```diff
@@ -266,6 +284,16 @@ sti::build::place_bins() {
   )
 }
 
+# sti::build::make_binary_symlinks makes symlinks for the sti
+# binary in _output/local/go/bin
+sti::build::make_binary_symlinks() {
+  if [[ -f "${STI_LOCAL_BINPATH}/s2i" ]]; then
+    for linkname in "${STI_BINARY_SYMLINKS[@]}"; do
+      ln -sf "${STI_LOCAL_BINPATH}/s2i" "${STI_LOCAL_BINPATH}/${linkname}"
```

Comment:
I got nasty here by copying the code used in [origin](https://github.com/openshift/origin/blob/master/hack/common.sh#L336-L342). I'm not even sure how to create windows symlinks, do you think we should care, knowing the fact that docker does not run on windows natively, as of yet at least?